packages/cli/src/ui/components/messages/ToolMessage.tsx

```diff
@@ -56,6 +57,7 @@ export const ToolMessage: React.FC<ToolMessageProps> = ({
   ptyId,
   config,
 }) => {
+  const { renderMarkdown: globalRenderMarkdown } = useUIState();
```

Comment:
nit: just call this render markdown. globalRenderMarkdown is an odd name that makes it sound like this is a global variable.